src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -104,6 +105,12 @@ public Loan changeDueDate(DateTime newDueDate) {
     return this;
   }
 
+  public Loan changeDueDateChangedByRecall(boolean newDueDateChangedByRecall) {
```

Comment:
As I understand it, there is only need to mark this as having happened, and not reset it. Hence, is there any need for a parameter here?

Reply:
When I was coding it initially, there was a question as to whether or not this should be reset if the loan is renewed after a cancelled recall request is made. You are correct that Cate has indicated that this is not possible now. I'll remove the parameter.